main.js

```diff
@@ -1,35 +1,164 @@
+const projectCard = document.getElementById('work-container');
+
+const cardData = [
+  {
+    id: 1,
+    image: 'Project-Images/ecommerce sit.PNG',
+    title: 'Clothing Store eCommerce Website',
+    technogies: ['Ruby on rails', 'CSS', 'HTML', 'JavScript'],
```

Comment:
- 👏🏻 👏🏻 Excellent work implements an array of objects to store JavaScript objects containing informations for each project. 😉  💯

Reply:
Dear @BenMukebo , 
thanks